types/body-parser/body-parser-tests.ts

```diff
@@ -1,35 +1,37 @@
 import * as http from 'http';
 import express = require('express');
-import {
```

Comment:
Why remove these import tests?

Reply:
They're valid. I've removed them to test and follow examples from project:
https://github.com/expressjs/body-parser#examples
(as users usually start that way).
I can amend this, to use exported names, let me know.

Reply:
It would be nice if we put the exported names back - it would be nice if they didn't break (although I can't imagine how they would.  :P)

Reply:
sure, np., done. Changes are squashed, just white-space related changes via Prettier.
Thanks!